wormhole/file_transport.go

```diff
@@ -200,10 +201,16 @@ func (t *fileTransport) connectViaRelay(otherTransit *transitMsg) (net.Conn, err
 
 	var conn net.Conn
 
+	connectTimeout := time.After(5 * time.Second)
+
```

Comment:
This and the next hunk were deleted recently. https://github.com/LeastAuthority/wormhole-william/commit/3c87f248fe09be9a631e4d9d4d1da1a5ad531d53